lib/resty/etcd.lua

```diff
@@ -27,6 +27,13 @@ local function etcd_version(opts)
     return ver.body
 end
 
+local function prequire(prefix, package, default)
+    local o, p = pcall(require, prefix .. package)
+    if not o then
+         return require(prefix .. default)
```

Comment:
it seems wrong, if the `pcall` is failed, this line will fail too

Reply:
just in case package is an invalid value and downgrade to the default one